src/ts/createView.tsx

```diff
@@ -35,6 +38,7 @@ const createView = (
   view.dom.parentNode!.replaceChild(wrapperElement, view.dom);
   wrapperElement.appendChild(view.dom);
   view.dom.insertAdjacentElement("afterend", overlayNode);
+  logger.info("Typerighter plugin starting");
```

Comment:
Do we still need this line?

Reply:
Oh, wait, I see what's happening – ignore 🤦

Reply:
(I'd favour removing it as soon as our logger is doing useful work elsewhere)

Reply:
Yes,  that's the intent.